bioconda_utils/bioconda_utils-conda_build_config.yaml

```diff
@@ -9,50 +9,46 @@
 # clear hard-coded default value for CONDA_BUILD_SYSROOT
 CONDA_BUILD_SYSROOT:
   - ""
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
+c_stdlib_version:
+  - 2.17                       # [linux]
+  - 11.0                       # [osx and arm64]
+  - 10.13                      # [osx and x86_64]
 MACOSX_DEPLOYMENT_TARGET:      # [osx]
   - 11.0                       # [osx and arm64]
-  - 10.9                       # [osx and x86_64]
-
-pin_run_as_build:
-  htslib:
-    max_pin: x.x
-  bamtools:
-    max_pin: x.x.x
-  r-base:
-    max_pin: x.x
-    min_pin: x.x
+  - 10.13                      # [osx and x86_64]
 
 bamtools:
-  - "2.5.1"
+  - "2.5.2"
```

Comment:
I've created a PR exactly for this few days ago - https://github.com/bioconda/bioconda-utils/pull/988

Reply:
Thanks, we'll just roll this into the new update.